src/assets/translations/en-us.yaml

```diff
@@ -4509,6 +4509,16 @@ troubleshooting:
       buttonText: Factory Reset
   needHelp: 'Still having problems? Start a discussion on the <a href="https://slack.rancher.io/">Rancher Users Slack</a> or <a href="https://github.com/rancher-sandbox/rancher-desktop/issues">Report an Issue</a>.'
 
+##############################
+### Integrations Page
+##############################
+integrations: 
+  darwin:
+    title: Supporting Utilities
+    description: Create symbolic links to tools in /usr/local/bin
+  windows:
+    title: WSL Integration
+
```

Comment:
Two quick things...

- For WSL can we add a description of something like, "Select Windows Subsystem for Linux (WSL) distros to expose Rancher Desktop's Kubernetes configuration in". People have asked what this does and a description would help.
- In the near future (after this PR) we will need to support linux in addition to darwin. It will be the same way.

Reply:
Thanks for the heads up on the lint error - fixed.

I also relocated the description to be part of the page header for both Windows and macOS since this is such a simple page. Screenshots have been updated.

The WSL Integration description that I landed on was `Expose Rancher Desktop's Kubernetes configuration to each selected Windows Subsystem for Linux (WSL) distro` - it just felt that it read a little better to swap the front and back half the description

Finally, I updated the OS check to assume `Supporting Utilities` is used if the os is not `win32`.